apps/coscrad-frontend/src/components/resources/playlists/playlist-detail.thumbnail.presenter.tsx

```diff
@@ -1,7 +1,18 @@
 import { ICategorizableDetailQueryResult, IPlayListViewModel } from '@coscrad/api-interfaces';
+import { ResourceDetailFullViewPresenter } from '../../../utils/generic-components/presenters/detail-views';
+import { EpisodePresenter } from './episode-presenter';
 
-export const PlaylistDetailThumbnailPresenter = (
-    _: ICategorizableDetailQueryResult<IPlayListViewModel>
-): JSX.Element => {
-    return <div>playlist</div>;
-};
+export const PlaylistDetailThumbnailPresenter = ({
+    name,
+    id,
+    episodes,
+}: ICategorizableDetailQueryResult<IPlayListViewModel>): JSX.Element => (
+    <ResourceDetailFullViewPresenter name={name} id={id}>
+        <div>
+            <h3> Episodes </h3>
+            {episodes.map((episode) => (
```

Comment:
We don't want this here. I recommend that you just add in a count "number of episodes: {episodes.length}".

Reply:
resolved